packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -130,13 +132,18 @@ export class InsuredBridgeL1Client {
       ]);
 
       for (const depositRelayedEvent of depositRelayedEvents) {
-        const relayData = {
+        const relayDataForDeposit = await this.bridgePools[depositRelayedEvent.returnValues.l1Token].methods
```

Comment:
hhmmm. I think we should not be doing this. this will make this method extremely slow as we will have async calls for each and every relay. is there no way for us to pull this out of the event? perhaps we need to modify the event to include this timestamp? ideally this method should run in constant time without needing to make extra calls per relay event.

Reply:
Great catch, we no longer need to modify the `relayData` to verify a relay's ancillary data! I removed all changes to this client.